modules/util/source-formatter/src/main/resources/sourcechecks.xml

```diff
@@ -1772,6 +1772,10 @@
 			<category name="Upgrade" />
 			<description name="Replace Liferay's RejectedExecutionHandler with Java's RejectedExecutionHandler" />
 		</check>
+		<check name="UpgradeSCSSNodeSassPatternsCheck">
+			<category name="Upgrade" />
+			<description name="Run code migration the division using of '/' by math.div and change the interpolation in SCSS file" />
```

Comment:
Hi @TamyrisTorres, maybe we could rephrase the description to 'Run code migration of Dart Sass deprecated patterns (e.g., the division operation using the '/' character, the interpolation syntax, etc.)' what you think?